### PR TITLE
foreskin

### PR DESCRIPTION
**Pull request by Acensti**
foreskin + job thanks pp

### CODE REVIEW

File: code/modules/jobs/job_types/chaplain.dm [hunk 1]
```diff
@@ -133,3 +133,4 @@
 	chameleon_extras = /obj/item/stamp/chap
 
 	id_trim = /datum/id_trim/job/chaplain
+datum/job/rabbi
```
Comment (SpaceLoveSs13) [inline code/modules/jobs/job_types/chaplain.dm:136]: Correct the typepath

File: code/modules/judaism/foreskin.dm [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/obj/item/organ/foreskin
+	name = "foreskin"
+	desc = "A small strip of skin harvested from a male. Certain cultures attribute mythical powers to these."
+	zone = BODY_ZONE_PRECISE_GROIN
```
Comment (SpaceLoveSs13) [inline code/modules/judaism/foreskin.dm:4]: define this first

File: code/modules/judaism/rabbi.dm [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+
+/obj/effect/landmark/start/rabbi
+	name = "Rabbi"
+	icon = 'ppstation/icons/jew.dmi'
+	icon_state = "jewspawn"
+
+/datum/job/rabbi
+	title = "Rabbi"
+	flag = RABBI
```
Comment (SpaceLoveSs13) [inline code/modules/judaism/rabbi.dm:48]: again define and correct the typepath

**Comment by SpaceLoveSs13** [comment]
fix all the missing indentations and define the code properly. Also modularize everything.

File: code/modules/judaism/foreskin.dm (hunk 2 above)
Comment (Acensti) [reply code/modules/judaism/foreskin.dm:4]: My child, Who sent you!.

**Pull request closed without merging**